src/components/NcPopover/NcPopover.vue

```diff
@@ -162,9 +208,10 @@ See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/
 </template>
 
 <script>
-import { warn } from 'vue'
+import { isRTL } from '@nextcloud/l10n'
```

Comment:
Should be already computed once per page
```suggestion
import { isRtl } from '../../utils/rtl.ts'
```